proto/prysm/v1alpha1/validator.proto

```diff
@@ -294,6 +294,16 @@ service BeaconNodeValidator {
         };
     }
 
+    // Server-side stream of all slots of valid blocks as they are received by
+    // the beacon chain node.
+    // DEPRECATED: This endpoint is superseded by the /eth/v1/events Beacon API endpoint
+    rpc StreamSlots(StreamSlotsRequest) returns (stream StreamSlotsResponse) {
```

Comment:
In the future if we want to enable people to use our validator with other clients... is there something else we can use in its place? it's still marked as deprecated

Reply:
yeah, the REST validator should hook to `eth/v1/events`  and use the BeaconAPI. This PR is a placeholder in the REST case.